frontend/src/layouts/Platform.vue

```diff
@@ -13,6 +13,10 @@
                                        :type="a.type" :message="a.message"
                                        :countdown="a.countdown || 3000" @close="clear($index)"></ff-notification-toast>
             </TransitionGroup>
+            <ff-dialog ref="dialog" :header="dialog.header" :kind="dialog.kind" :disable-primary="dialog.disablePrimary" :confirm-label="dialog.confirmLabel" @cancel="clearDialog" @confirm="dialog.onConfirm">
+                <p v-if="dialog.text">{{ dialog.text }}</p>
+                <div class="space-y-2" v-html="dialog.html" />
```

Comment:
```suggestion
                <div class="space-y-2" v-html="dialog.html"></div>
```
Self closing DIV is not w3c valid